androidApp/src/main/java/com/mbta/tid/mbta_app/android/stopDetails/AlertCard.kt

```diff
@@ -76,23 +76,13 @@ fun AlertCard(
             AlertIcon(
                 alertState = alert.alertState,
                 color = color,
-                modifier =
-                    Modifier.size(iconSize)
-                        .align(
-                            if (spec == AlertCardSpec.Elevator) Alignment.Top
-                            else Alignment.CenterVertically
-                        )
+                modifier = Modifier.size(iconSize).align(Alignment.CenterVertically)
             )
             Text(
                 formattedAlert.alertCardHeader(spec),
                 Modifier.weight(1f),
                 style =
-                    if (
-                        spec == AlertCardSpec.Major ||
-                            (alertSummary == null && spec != AlertCardSpec.Elevator)
-                    )
-                        Typography.title2Bold
-                    else Typography.body
+                    if (spec == AlertCardSpec.Major) Typography.title2Bold else Typography.callout
```

Comment:
@BrandonTR maybe worth confirming with design, but based on [this item](https://www.figma.com/design/OVO5Z0eYUtnnRvbxsIRpO3/App-QA?node-id=1859-1025&t=xsfBgU2kdynaEG0C-0) I wonder if secondary alerts should be calloutBold when not using the AlertSummary format.